api-docs/client-follow-up.md

```diff
@@ -0,0 +1,91 @@
+# Client follow up note
+
+The client follow up documents and updates any calls with the client and documents the interaction as well as set up a date to follow up with the client.
+
+## Creating a follow up note on a client
+
+### Request
+
+```http
+POST /api/clients/:clientId/follow-up
+```
+
+```json
+{
```

Comment:
So it's okay if the **PATCH** request doesn't have the `id` property since it is in the params, right? And should the response json also have the id property for the **POST** request?

Reply:
Ah I put the comment here on the wrong line. The `id` should always be present in the Response body, but it doesn't need to be present in the request body. For POST request body, there's no id yet. And for PATCH, it's in the URL params like you mentioned.